bin/webpack-configs.js

```diff
@@ -367,7 +365,39 @@ const getFrontConfig = ( options = {} ) => {
 			concatenateModules:
 				isProduction && ! process.env.WP_BUNDLE_ANALYZER,
 			splitChunks: {
-				automaticNameDelimiter: '--',
+				cacheGroups: {
+					default: false,
```

Comment:
In theory, this group would group common shared internal code into a single file, however, it resulted in a very awkward files with tilds and very long folder names.

I think I can make this better by having a minSize here and more controlled name function. One issue stands in which a split file would end up being empty but that's up for testing first.